src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -38205,12 +38202,9 @@ interface I19
                 // (70,20): error CS0542: 'I15': member names cannot be the same as their enclosing type
                 //     static partial I15();
                 Diagnostic(ErrorCode.ERR_MemberNameSameAsType, "I15").WithArguments("I15").WithLocation(70, 20),
-                // (74,5): error CS0267: The 'partial' modifier can only appear immediately before 'class', 'struct', 'interface', or 'void'
+                // (74,13): error CS1585: Member modifier 'static' must precede the member type and name
```

Comment:
I have a solution for this to be reported as invalid modifier. I intend to try it out when I'm doing the the work for the `async` modifier after this.

Reply:
BTW would it be ok to move langver check out of parser to a later pass?